### DISCUSSION TITLE
Equality Blockchain not will syn for Version 1.1.714

### DISCUSSION THREAD

**Discussion by Akkarawat0510** [Chia-Network/chia-blockchain#7119] [Support]
Please advise to support and update new version 
issue not will syn and found proof (not get the reward)

Also 
Do you have Roadmap(Web) or Discord to see the update for this coin

Thank you

**Comment by loppefaaret** [comment]
we use keybase a chat service:
https://keybase.io/team/chia_network.public

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
